src/_gettsim/wohngeld/wohngeld.py

```diff
@@ -92,6 +96,30 @@ def basisformel(
     Note: This function is not a direct target in the DAG, but a helper function to
     store the code for Wohngeld calculation.
 
+    """
+    a = params.a.look_up(anzahl_personen)
+    b = params.b.look_up(anzahl_personen)
+    c = params.c.look_up(anzahl_personen)
+    out = xnp.maximum(
```

Comment:
```suggestion
    max_aus_formel_m = xnp.maximum(
```

Same in function just below. There, please change to:

```suggestion
    max_aus_formel_m = zusatzbetrag_nach_haushaltsgröße + xnp.maximum(
```

Makes it easier to read.